Magento2/ruleset.xml

```diff
@@ -522,6 +522,31 @@
     <rule ref="Squiz.WhiteSpace.SuperfluousWhitespace.EmptyLines">
         <severity>0</severity>
     </rule>
+    <rule ref="Magento2.GraphQL.ValidTypeName">
```

Comment:
Please respect rules order in the `ruleset` - by severity -> by name.

Reply:
Fixed in https://github.com/magento/magento-coding-standard/pull/141/commits/c539ed44c3b93284d9bc24e5e58c0271cbaa1d40